tools/small_read_size_histograms/size_histogram.xml

```diff
@@ -1,15 +1,14 @@
-<tool id="Size_histogram" name="Generate size histograms from alignment files" version="0.9.9">
-    <description>from sRbowtie aligment</description>
+<tool id="artbio_size_histogram" name="Generate read size histograms" version="1.0.0">
+    <description>from alignment files</description>
     <requirements>
-        <requirement type="package" version="1.1.2">bowtie</requirement>
-        <requirement type="package" version="0.8.3">pysam</requirement>
-        <requirement type="package" version="3.1.2">R</requirement>
-        <requirement type="package" version="2.14">biocbasics</requirement>
-        <requirement type="package" version="1.9">numpy</requirement>
-        <requirement type="package" version="2.0.0=r3.2.2_0a">r-gridextra</requirement>
-        <requirement type="package" version="0.6_26=r3.2.2_2a">r-latticeextra</requirement>
+        <requirement type="package" version="1.2.0=py27_0">bowtie</requirement>
+        <requirement type="package" version="0.11.2.1=py27_0">pysam</requirement>
```

Comment:
You said you wanted to slowly change the python version of the scripts in Galaxy to python3.
Wouldn't it be better to do it here instead of leaving it in python2 then changing it to python3 an other time?
(I ask this because this question applies to mircounts too since it uses pysam of python2 right now)